floss/main.py

```diff
@@ -151,6 +151,17 @@ def make_parser(argv):
         help="path to sample to analyze",
     )
 
+    language_group = parser.add_argument_group("language arguments")
+    language_group.add_argument(
+        "-e",
+        "--extract",
```

Comment:
out my proposals in issue 830 I like `extract` the least, @williballenthin thoughts?

Reply:
Actually, I was considering adding the "--language" option, but it seems that the "-l" flag is already in use. Do you have any suggestions?

Reply:
Regarding the flag options, what are your thoughts on using both '-g' and '--language' together? Do you think it would be feasible or could it lead to confusion?

Reply:
We could just use `--language` without a short argument.

Reply:
yeah, i think we should reserve short arguments for those that will be used very often or are very obvious/expected. `--language` seems like an advanced feature, so lets restrict it to a long argument.

Reply:
good, and then add it to the advanced options, like `"--format"`